server/Server.Domain/Commands/SetupCustomerCommand.cs

```diff
@@ -1,13 +1,14 @@
 namespace Server.Domain.Commands;
 
-using MediatR;
 using Microsoft.EntityFrameworkCore;
 using Server.Domain;
 using Server.Domain.Models;
 
 public class SetupCustomerCommand : ICommand<BookRepoContext>
 {
     public required string Id { get; init; }
+    public List<string>? BookshelvesNames { get; init; }
```

Comment:
Generally prefer a flat structure for a command pattern. Maybe this logic would make more sense being served via a provider or split out into multiple commands. Its functional for now, but room for improvements